### PR TITLE
setting install prefix in configure script

### PR DESCRIPTION
**Pull request by mm-s**
configure was not passing CMAKE_INSTALL_PREFIX to cmake

### CODE REVIEW

**Review by segfaultxavi — COMMENTED** [review]
Have you tested it? Doesn't this set an empty prefix when the envvar is not set?

**Comment by mm-s** [comment]
> Have you tested it? Doesn't this set an empty prefix when the envvar is not set?

Yes I tested it when the envvar is not set. In that case it ends up using the /usr/local default destination as expected.

**Comment by segfaultxavi** [comment]
In that case I have no further comments, except that you should be consistent using double backticks everywhere (``).
The text looks fine to me but somebody else should review the functional changes.

File: docs/BUILD-manual.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ sudo ninja install
 ```
 
 > **NOTE:**
-> You can change the default installation location passing ``-DCMAKE_INSTALL_PREFIX=...`` to ``cmake``. In this case you might not require ``sudo``.
+> You can change the default installation location running step 2 with the ``CMAKE_INSTALL_PREFIX`` environment variable set, e.g. ``CMAKE_INSTALL_PREFIX=... scripts/configure-manual.sh``. In this case you might not require ``sudo``.
```
Comment (Jaguar0625) [inline docs/BUILD-manual.md:74]: *by* running ?

**Comment by Jaguar0625** [comment]
can you please rebase on main?

**Comment by mm-s** [comment]
> can you please rebase on main?

rebased

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**